go.mod

```diff
@@ -1,48 +1,53 @@
 module github.com/IBM/ibm-block-csi-operator
 
-go 1.13
+go 1.15
```

Comment:
why not `1.15.14`?

Reply:
I think you put there the major version of your go and it more for description as far as I know